net/gost/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/ginuerzh/gost v2.11.1
+revision            0
+
+categories          net
+license             MIT
+maintainers         {outlook.de:judaew @judaew}
```

Comment:
Do you want to enforce that just you and only you can make changes to this port?

If not, then you should include `openmaintainer`.

Reply:
Fix